### ISSUE TITLE
EdlessScrolling on JokesList

### ISSUE THREAD

**Issue by MrCatDog** [MrCatDog/ChuckJokes#2]
RecyclerView must update items while scrolling.

**Comment by MrCatDog** [comment]
I will rework RecyclerAdapter for this. I should add setter and getter for data and some mechanism for building new elements.
Only after that i will add ScrollListener.

**Comment by MrCatDog** [comment]
I guess i have progress here: RecyclerAdapter now have reciveData method, which call DataReciver, takes new data sets and redraw RecyclerAdapter. I should rewrite dataset update to increase it, not overwrite.

**Comment by MrCatDog** [comment]
Endless scrolling is on work. I will add another issue for updating RecyclerView.
